src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -383,6 +389,14 @@ public boolean equals(Object obj) {
             return false;
         }
 
+        if (questionDescription == null) {
+            if (other.questionDescription != null) {
```

Comment:
Hmmm I guess I got it confused. Anyway, this would be best as an XOR operation, i.e `(questionDescription == null ^ other.questionDescription == null)`.